app/resources/api/v3/public/card_resource.rb

```diff
@@ -18,6 +19,10 @@ class Api::V3::Public::CardResource < JSONAPI::Resource
         has_many :card_subtypes
         has_many :printings
 
+        def latest_printing_id
+          @model.printings[-1]['id']
```

Comment:
is this guaranteed to be sorted reverse chronologically?

Reply:
I believe it's guaranteed provided the json source is in chronological order; which is to say I'm 98% sure it works now but there's no guarantee it will continue to. I'll merge it for now but note that I need to clean this up later.